### PR TITLE
Uniformized the calling methods for integration tests

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Reasoning behind the pull request
- 
- 
- 
  
## Proposed changes
- 
- 
- 

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: integrationTests/vm/wasm/upgrades/upgrades_test.go [hunk 1]
```diff
@@ -19,6 +15,10 @@ import (
 )
 
 func TestUpgrades_Hello(t *testing.T) {
+	if testing.Short() {
```
Comment (andreibancioiu) [inline integrationTests/vm/wasm/upgrades/upgrades_test.go:18]: These tests are quick.

File: integrationTests/vm/wasm/transfers/transfers_test.go [hunk 2]
```diff
@@ -13,6 +11,10 @@ import (
 )
 
 func TestTransfers_DuplicatedTransferValueEvents(t *testing.T) {
+	if testing.Short() {
```
Comment (andreibancioiu) [inline integrationTests/vm/wasm/transfers/transfers_test.go:14]: These tests are quick.

File: integrationTests/vm/wasm/erc20/erc20_test.go [hunk 3]
```diff
@@ -10,9 +8,8 @@ import (
 )
 
 func Test_C_001(t *testing.T) {
-	// TODO reinstate test after Wasm VM pointer fix
 	if testing.Short() {
-		t.Skip("cannot run with -race -short; requires Wasm VM fix")
+		t.Skip("this is not a short test")
```
Comment (andreibancioiu) [inline integrationTests/vm/wasm/erc20/erc20_test.go:12]: These tests are quick.

File: integrationTests/vm/wasm/badcontracts/badcontracts_test.go [hunk 4]
```diff
@@ -11,9 +9,8 @@ import (
 )
 
 func Test_Bad_C_NoPanic(t *testing.T) {
-	// TODO reinstate test after Wasm VM pointer fix
 	if testing.Short() {
-		t.Skip("cannot run with -race -short; requires Wasm VM fix")
+		t.Skip("this is not a short test")
```
Comment (andreibancioiu) [inline integrationTests/vm/wasm/badcontracts/badcontracts_test.go:13]: These tests are quick (and it's good to have them run on GH MacOS, as well).

**Review by andreibancioiu — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**